python/cugraph/structure/graph.py

```diff
@@ -103,6 +104,51 @@ def clear(self):
         self.adjlist = None
         self.transposedadjlist = None
 
+    def add_nodes_from(self, nodes, bipartite='nodes'):
```

Comment:
Would it be better to use:  `bipartite=None` as the final parameter.  This introduces a limitation that a user can't name one of their type sets "none".

Reply:
Changed to bipartite = None. If this argument is not provided, the nodes are simply assumed to be all the nodes present in the graph.